src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -4046,6 +4062,61 @@ public Derived() : this(0)
         );
     }
 
+    [Fact, CompilerTrait(CompilerFeature.NullableReferenceTypes)]
+    [WorkItem(61718, "https://github.com/dotnet/roslyn/issues/61718")]
+    public void RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_09C()
+    {
+        var @base = $$"""
+            #nullable enable
+            public class Base
+            {
+                private string _field1 = null!;
+                public required string Prop1 { get => _field1; set => _field1 = value; }
+
+                protected Base() {}
+            }
+            """;
+
+        var derived = $$"""
+            using System.Diagnostics.CodeAnalysis;
+            #nullable enable
+
+            public class Derived : Base
+            {
+                private string _field2 = null!;
+                public required string Prop2 { get => _field2; set => _field2 = value; }
+
+                [SetsRequiredMembers]
+                public Derived(int unused) : base()
+                {
+                }
+
+                public Derived() : this(0)
```

Comment:
Was there something specific being demonstrated by the absence of SetsRequiredMembers here?

Reply:
Mainly the error about it.